package-res/reportviewer/paginationTemplate.html

```diff
@@ -34,8 +34,8 @@
 <div>
   <div class="page-control">
     <div class="page-control-nav">
-      <span id="left" onclick="left()"><&nbsp;</span>
-      <span id="right" onclick="right()">>&nbsp;</span>
+      <span id="left"><&nbsp;</span>
+      <span id="right">>&nbsp;</span>
```

Comment:
Nope, that line does not look correct. There are two >> characters on the right-span.